loop_rate: 500.0 # the frequency at which to step sim in h
gravity: -9.81 # earth gravity in m/s^2
#max_torque: 100.0 # the maximal torque to apply to the robot joints in velocity mode

robots:
  franka:
    - module: pybullet_ros.plugins.control
      class: Control
    - module: pybullet_ros.plugins.joint_state_publisher
      class: JointStatePublisher
